rust-compile.el

```diff
@@ -34,6 +34,12 @@ See `compilation-error-regexp-alist' for help on their format.")
   "Specifications for matching code references in rustc invocations.
 See `compilation-error-regexp-alist' for help on their format.")
 
+(defvar rustc-panics-compilation-regexps
+   (let ((re "thread '[^']+' panicked at \\(\\(.*\\):\\([0-9]+\\):\\([0-9]+\\)\\)"))
```

Comment:
I think you should use the `rustc-compilation-location` variable already in scope.
Something like this
```
(let ((re (concat "thread '[^']+' panicked at " rustc-compilation-location)))
    (cons re '(2 3 4 nil 1)))
    ```

Reply:
Done, you were right!